Update cloudhub-deploy-options.adoc (#83)
mulesoft/docs-partner-manager: modules/ROOT/pages/cloudhub-deploy-options.adoc

include::_attributes.adoc[]
endif::[]

Partner Manager message flows are initially deployed by default to the CloudHub region specified in the default

> settings of Access Manager's business group. Subsequent deployments are deployed to the same region as the first deployment.

Reviewer: Does the second sentence make sense? 
Is there a miswording here?

If the initial deployment goes to Region A, let's call it, and subsequent deployments go to "the same region as the first deployment", which is Region A, then what is the point of distinguishing between initial and subsequent (if they are deployed to the same region)?

Reviewer: After reading further, I wonder if what you really mean here is that all message flows are deployed to the default region and that if you change the default for a deployment, that change does not persist in subsequent deployments (which continue to use the default unless individually changed).

If this is what you mean, reword to clarify. Don't necessarily use MY wording above, which is not very refined....

To access the default deployment settings for your organization, navigate to *Access Management > Partner Manager*.

> You can configure the following Partner Manager Settings for CloudHub deployment.

Reviewer: How does this sentence relate to the previous?
Are we saying that after you access the defaults, you can customize the following settings? clarify
Additionally, colon before list:
```suggestion
You can configure the following Partner Manager settings for CloudHub deployment:
```

Reviewer: Also, search this PR for "configur" and evaluate whether it should be replace with "customiz", because I think that customization makes more clear that you are altering a default config that already exists.

* Runtime version

> The default and recommended option is `Use latest version`, which deploys Partner Manager new message flows on the latest Mule runtime version.

Reviewer: ```suggestion
The default and recommended option is `Use latest version`, which deploys new Partner Manager message flows by using the latest Mule runtime version.
```

> If you want the applications deployed to a specific older version, choose the appropriate version from the menu.

Reviewer: ```suggestion
If you want the applications deployed to another version, choose it from the menu.
```

* CloudHub region

The default option is retrieved from the business group’s default region setting for Runtime Manager in Access Manager.
If you want the B2B integration flows from Partner Manager to be deployed to a different region,
select it from the drop-down menu prior to deploying your first message flow.

> Subsequent deployments get deployed to the same region as the first deployment.

Reviewer: Again, are we really saying that each time you change the default, that change applies only to that one deployment and that the default persists unless you change individual deployment settings?
Additionally:
```suggestion
Subsequent deployments are deployed to the default.
```
OR...are we saying that if you change the setting from the default once, that change applies ONLY to that one time and all subsequent deployments use the default?
If that is what we mean, reword to clarify, something like...:

> To change the default region moving forward, you must undeploy and redeploy all the previously deployed message flows.

Reviewer: ```suggestion
To change the default region and have it persist as the default, you must undeploy and redeploy any previously deployed message flows.
```

For more information refer to xref:configure-message-flows.adoc[Configure Message Flows].

* Anypoint MQ region

> This is automatically chosen based on the CloudHub region and cannot be changed.

Reviewer: ```suggestion
This is automatically assigned, based on the CloudHub region; you cannot change it.
```

* Automatically restart app when not responding

> If your system allows, you can set this to restart Partner Manager if it does not respond.

Reviewer: What determines your system "allowing" this setting? Licensing? Admin settings?

Reviewer: How about 
```suggestion
If this feature is enabled, your system automatically restarts Partner Manager if it does not respond.
```

For more information, refer to https://docs.mulesoft.com/runtime-manager/worker-monitoring#enable-automatic-restarts[Enable Automatic Restarts].

> * vCore Settings

Reviewer: ```suggestion
* vCore settings
```

> Configure these at the Sandbox or Production environment level.

Reviewer: ```suggestion
Configure these in either sandbox or production environments.
```
Per our style ref. and terms list, these are lowercase.

> The `Document Processing` and `API Replication` Services are core components of the Partner Manager workflow,

Reviewer: ```suggestion
The `Document Processing` and `API Replication` services are core components of the Partner Manager workflow,
```

> requiring a minimum of 1 vCore each. Each unique trading partner and internal endpoints get deployed to isolated workers - requiring a minimum of 0.1 vCore each.

Reviewer: ```suggestion
requiring a minimum of one vCore each, and each unique trading partner and internal endpoint is deployed to isolated workers, requiring a minimum of 0.1 vCore each.
```

> If you require assistance with sizing, please contact your Account representative.

Reviewer: Are we mentioning account reps anymore, or has that changed to just Support, a la Salesforce?
```suggestion
If you require assistance with sizing, contact your account representative.
```
